sdk/servicebus/Azure.Messaging.ServiceBus/README.md

```diff
@@ -182,7 +182,7 @@ string body = receivedMessage.Body.ToString();
 Console.WriteLine(body);
 ```
 
-### Send and receive a batch of messages
+### Send a batch of messages
```

Comment:
We'll also want to update the snippet name for sending on L189 so that it reflects that it is not receiving anything.

Reply:
Reverted to the original markdown file, just added the missing code which is supposed to receive a batch of messages.